script.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function(){
+
+	// $('span').on('click', function(){
+	// 	alert('clicked!');
+	// });
+
+	// Put something on list
+	$('#name-form').on('submit', function(e){
+		e.preventDefault();
+		var $name = $('#name-form input').val();
+		console.log($name);
+
+		$('ol').append('<li id="item">' + $name + ' <span id="delete">&#9986;</span>  </li>').sortable();
```

Comment:
Before the end of the function, be sure to clear the value of the input form so that it's easier for the user to type in the next item on the agenda.

```
$('#name-form input').val() = '';
```